pkgs/tools/misc/lazydocker/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  name = "lazydocker-${version}";
```

Comment:
```suggestion
  pname = "lazydocker";
```

And renove `rec`

Reply:
> ```suggestion
>   pname = "lazydocker";
> ```

:+1: 

> And renove `rec`

If I remove `rec` from the argument to `buildGoModule`, I won't anymore be able to use
```nix
"v${version}"
```
for `rev` below.

Please advise.

Reply:
Oh right, `rec` stays then.

Reply:
switched to `pname` in 1658e84